docs/Performance.md

```diff
@@ -69,3 +69,41 @@ Measured memory usage (in MB) for particular Events Per Second (EPS) average mes
 | **1000** | 121.6  | 126.75 | 127.94 | 140.11 | 106.82 |
 | **1500** | 128.54 | 131.9  | 137.69 | 95.21  | 113.89 |
 | **2000** | 130.62 | 125.27 | 144.59 | 98.62  | 134.61 |
+
+## Fine Tuning
```

Comment:
Not sure if this should be he same header level like `logs`
Maye we should put it in separate file, IDK

Reply:
We should definitely reorganize these to put test results in under one `##` section and fine tuning in a different `##` section, IMHO.